Add Retool docs (#1705)
ClickHouse/clickhouse-docs: docs/en/integrations/no-code/retool.md

Select "JDBC" from the list of available connectors:

<img src={require('./images/retool_02.png').default} class="image" alt="Choosing JDBC connector" style={{width: '75%', 'background-color': 'transparent'}}/>
<br/>

In the setup wizard, make sure you select `com.clickhouse.jdbc.ClickHouseDriver` as the "Driver name":

<img src={require('./images/retool_03.png').default} class="image" alt="Selecting the right driver" style={{width: '75%', 'background-color': 'transparent'}}/>
<br/>

Fill in your ClickHouse credentials in the following format: `jdbc:clickhouse://HOST:PORT/DATABASE?user=USERNAME&password=PASSWORD`. 
If your instance requires SSL or you are using ClickHouse Cloud, add `&ssl=true` to the connection string, so it looks like `jdbc:clickhouse://HOST:PORT/DATABASE?user=USERNAME&password=PASSWORD&ssl=true`

<img src={require('./images/retool_04.png').default} class="image" alt="Specifying your credentials" style={{width: '75%', 'background-color': 'transparent'}}/>
<br/>

After that, test your connection:

<img src={require('./images/retool_05.png').default} class="image" alt="Testing your connection" style={{width: '75%', 'background-color': 'transparent'}}/>
<br/>

> After that, you should be able to proceed to your app using your ClickHouse resource.

Reviewer: To avoid 2 `after that`  in the row
```suggestion
Now, you should be able to proceed to your app using your ClickHouse resource.
```
